helix-view/src/register.rs

```diff
@@ -210,17 +213,17 @@ impl Registers {
         }
     }
 
-    pub fn clipboard_provider_name(&self) -> Cow<str> {
-        self.clipboard_provider.name()
+    pub fn clipboard_provider_name(&self) -> Cow<'_, str> {
```

Comment:
Let's switch this to return a `String` since the `DynAccess` already forces us to own the value and the only caller immediately converts the value to a String anyway